packages/create-instance/create-functional-component.js

```diff
@@ -4,35 +4,35 @@ import { compileToFunctions } from 'vue-template-compiler'
 import { throwError } from 'shared/util'
 import { validateSlots } from './validate-slots'
 
-function createFunctionalSlots (slots = {}, h) {
-  if (Array.isArray(slots.default)) {
-    return slots.default.map(h)
-  }
+function startsWithTag (str) {
```

Comment:
`startsWithTag` is also defined in add-slots.js, I guess it could be extracted to some helper

Reply:
Right! Actually, we could use the `add-slots` files as well.

Let me refactor it :+1: